ballerina/CompilerPlugin.toml

```diff
@@ -3,4 +3,4 @@ id = "xmldata-compiler-plugin"
 class = "io.ballerina.stdlib.xmldata.compiler.XmldataCompilerPlugin"
 
 [[dependency]]
-path = "../compiler-plugin/build/libs/xmldata-compiler-plugin-2.4.1.jar"
+path = "../compiler-plugin/build/libs/xmldata-compiler-plugin-2.4.2-SNAPSHOT.jar"
```

Comment:
This has to be reverted. We have to rely on a released version, this is managed by a Gradle plugin.